scripts/ci/github_actions/driver/wait_for_driver_to_be_ready.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+set +o pipefail
+
+driver_is_ready=false
+actual_driver_running_time_in_seconds=0
+minimum_driver_running_time_in_seconds=10
+while [ "$(kubectl get pod -A -l csi | grep controller | wc -l)" -eq 0 ]; do
+  echo "The controller is not deployed"
```

Comment:
why specifically controller?

Reply:
just wanted to check that the controller/node was deployed and start waiting for all the pods to be ready, needed only one of them, so choosed controller

Reply:
still not following - why needed only one of them? what if not all pods are ready?

Reply:
I am not checking that only one of them is ready or not, I just checking that one of them is started to be deployed and then checking if they are both ready

Reply:
why does it matter if one of them is started to be deployed? can't we just check if they are both ready?

Reply:
because I don't want to check specific pods, I want to check all the pods under `-l csi`, and if this will only be the check, the script can finish running before the driver pods were deployed

Reply:
then what if there is a bug in the operator-k8s interaction and the other driver component wasn't started to be deployed?

Reply:
ok I can add also a check if the node has started, but we will need to maintain it if there will be a change in the driver pods (if we will add one or we will remove one )

Reply:
sounds good to me. not sure how we can avoid it

Reply:
done